File: app/src/main/java/com/cradle/neptune/view/EducationActivity.java
package com.cradle.neptune.view;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.cradle.neptune.R;

public class EducationActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_education);
        setupOnCLickMethods();
        if (getSupportActionBar() != null) {
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
            getSupportActionBar().setTitle("Help");
        }
    }

    private void setupOnCLickMethods() {
        CardView postercard = findViewById(R.id.communityPosterView);
        postercard.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent =new Intent(EducationActivity.this,PdfViewActivity.class);
                intent.putExtra("poster","education_community_poster.pdf");
                startActivity(intent);
            }
        });

        CardView videoCard = findViewById(R.id.videoCardview);
        videoCard.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(HelpActivity.makeIntent(EducationActivity.this));
            }
        });

        CardView clinicPoster = findViewById(R.id.clinicPosterView);
        clinicPoster.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent =new Intent(EducationActivity.this,PdfViewActivity.class);
                intent.putExtra("poster","education_clinic_poster.pdf");
                startActivity(intent);
            }
        });
    }


    @Override
    public boolean onSupportNavigateUp() {
        onBackPressed();
        return true;
    }
}

File: app/src/main/java/com/cradle/neptune/view/EducationActivity.kt
package com.cradle.neptune.view

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.cardview.widget.CardView
import com.cradle.neptune.R

class EducationActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_education)
        setupOnCLickMethods()
        if (supportActionBar != null) {
            supportActionBar?.setDisplayHomeAsUpEnabled(true)
            supportActionBar?.title = getString(R.string.education_activity_title)
        }
    }

    private fun setupOnCLickMethods() {
        val communityPosterCard = findViewById<CardView>(R.id.communityPosterView)
        communityPosterCard.setOnClickListener {
            val intent = PosterViewActivity.makeIntent(
                this@EducationActivity,
                R.drawable.educational_community_poster
            )
            startActivity(intent)
        }
        val videoCard = findViewById<CardView>(R.id.videoCardview)
        videoCard.setOnClickListener {
            startActivity(VideoActivity.makeIntent(this@EducationActivity))
        }
        val clinicPosterCard = findViewById<CardView>(R.id.clinicPosterView)
        clinicPosterCard.setOnClickListener {
            val intent = PosterViewActivity.makeIntent(
                this@EducationActivity,
                R.drawable.educational_clinic_poster
            )
            startActivity(intent)
        }
    }

    override fun onSupportNavigateUp(): Boolean {
        onBackPressed()
        return true
    }
}
